python-sdk/src/langwatch/experiment/platform_run.py

```
@@ -144,6 +144,53 @@ def print_summary(self, exit_on_failure: Optional[bool] = None) -> None:
+        status_icon = {"completed": "OK", "failed": "FAIL", "stopped": "STOPPED"}.get(
+            self.status, self.status
+        )
+        return (
+            f"ExperimentRunResult("
+            f"status={status_icon}, "
+            f"passed={self.passed}, failed={self.failed}, "
+            f"pass_rate={self.pass_rate:.1f}%, "
+            f"duration={self.duration / 1000:.1f}s)"
+        )
+
+    def to_dataframe(self) -> "pd.DataFrame":
```

Ruff (0.15.10)

[error] Undefined name `pd`

(F821)